update intro to Previews (#4836)
Particular/docs.particular.net: previews/index.md

---
title: Platform previews
reviewed: 2020-05-20
---

Previews allow Particular Software to develop new features more quickly while better addressing user needs.

Before deciding to launch a Preview, a research process is undertaken to determine which features best address the needs of users to help ensure the success of a Preview.

> Previews typically run for several weeks and their success is determined by how much the feature is used in production. If successful, it will translate into the adoption into the Platform. Previews that are insufficiently used in production will be converted into an open community project.

Reviewer: I've slightly rephrased this part to
1. Remove time commitment. For some previews, we might decide to run longer and should not overcommit.
1. Community projects are all open-source software. Removed the emphasis on open.

```suggestion
Previews typically run for some period of time and their success is determined by customer adoption. If successful, it will translate into the adoption into the Platform. Previews that are insufficiently used in production will be converted into a community project.
```
